apps/hire/src/components/candidate-form.tsx

```
@@ -0,0 +1,194 @@
+            name="email"
+            children={(field) => (
+              <field.FieldInput type="email" label="Email" />
+            )}
+          />
+          <form.AppField
+            name="phone"
+            children={(field) => <field.FieldInput type="tel" label="Phone" />}
+          />
+          <form.AppField
+            name="seniorityId"
+            children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)